src/base.js

```diff
@@ -21,12 +23,12 @@ const Handler = new Augur.Handler(config, {
   clientOptions: {
     disableEveryone: true,
     ws: {
-      intents: 9474,
+      // intents: 9474
```

Comment:
This initially caused crashes on my end, I'll re-enable it

Reply:
Fortunately, Discord has said they're not enforcing Intents just yet due to some miscommunications about it.